src/core/build_env.go

```diff
@@ -161,51 +148,28 @@ func withUserProvidedEnv(target *BuildTarget, env BuildEnv) BuildEnv {
 
 // TestEnvironment creates the environment variables for a test.
 func TestEnvironment(state *BuildState, target *BuildTarget, testDir string) BuildEnv {
-	env := TargetEnvironment(state, target)
+	env := RuntimeEnvironment(state, target, path.IsAbs(testDir), true)
 	resultsFile := path.Join(testDir, TestResultsFile)
-	abs := path.IsAbs(testDir)
 
 	env = append(env,
 		"TEST_DIR="+testDir,
 		"TMP_DIR="+testDir,
 		"TMPDIR="+testDir,
+		"HOME="+testDir,
```

Comment:
Are these not part of the runtime env?

Reply:
They are part of a sandboxed runtime environment. At least that's the approach I took which allows me to use `RuntimeEnvironment` for `plz run`